test/truffle/docker/fedora/Dockerfile

```diff
@@ -2,6 +2,19 @@ FROM fedora:26
 
 MAINTAINER chris.seaton@oracle.com
 
+# The native image tool uses gcc and zlib.h to figure out struct layouts and link
+RUN dnf install -y gcc zlib-devel
+
+# Dependencies for using openssl
+RUN dnf install -y compat-openssl10-devel
+
+# Dependencies for building C extensions
+RUN dnf install -y clang llvm
+
+# Dependencies for building and using C++ extensions
+RUN dnf install -y libcxx-devel
+RUN dnf install -y libcxx
```

Comment:
Why moving them up, even above the locale?
This look weird because there is a `ENV` in the middle of `dnf install` commands.
I think they should stay below as they were, just split with more details.
Same in other Dockerfile.

Reply:
I see, I still think environment should be set first so, e.g., all `dnf install` commands run under the same environment.
But I'll leave the decision to you.

Reply:
Moving the locale to the top.